AUTHORS

```diff
@@ -0,0 +1,15 @@
+# This is the list of sv-tests's significant contributors.
```

Comment:
Are there more people who need to be added here now?

Reply:
It currently lists people with 5+ commits (excluding people working for Antmicro or Google who are not listed directly)

Reply:
Great!